src/GitHook/Command/FileCommand/PreCommit/PhpMd/PhpMdCheckConfiguration.php

```diff
@@ -34,4 +36,34 @@ public function getMinimumPriority(): int
 
         return (isset($this->config['priority'])) ? $this->config['priority'] : $defaultPriority;
     }
+
+    /**
+     * @return string
+     */
+    public function getRulesetPath(): string
+    {
+        return $this->config['rulesetPath'] ?? $this->getDefaultRulesetPath();
+    }
+
+    /**
+     * @throws \Exception
+     *
+     * @return string
+     */
+    protected function getDefaultRulesetPath(): string
+    {
+        $defaultRuleSetPathsByPriority = [
+            'config/phpmd-ruleset.xml',
+            'vendor/spryker/spryker/Bundles/Development/src/Spryker/Zed/Development/Business/PhpMd/ruleset.xml',
+            'vendor/spryker/development/src/Spryker/Zed/Development/Business/PhpMd/ruleset.xml',
+        ];
+
+        foreach ($defaultRuleSetPathsByPriority as $ruleSetPath) {
+            if (file_exists(PROJECT_ROOT . DIRECTORY_SEPARATOR . $ruleSetPath)) {
+                return $ruleSetPath;
+            }
+        }
+
+        throw new Exception('Ruleset file was not found in any location: ' . PHP_EOL . implode(PHP_EOL, $defaultRuleSetPathsByPriority));
```

Comment:
This look misplaced in the config, I'd rather have this logic in a model because it looks a lot like business logic to me.

Config should only have default paths as an array, then the detection and exception can move to the model.